Prevent extensions from using setUninstallURL due to potential abuse of this feature
---
diff --git a/toolkit/components/extensions/parent/ext-runtime.js b/toolkit/components/extensions/parent/ext-runtime.js
index abc30a2e13..22f03e99a8 100644
--- a/toolkit/components/extensions/parent/ext-runtime.js
+++ b/toolkit/components/extensions/parent/ext-runtime.js
@@ -354,7 +354,7 @@ this.runtime = class extends ExtensionAPIPersistent {
         },
 
         setUninstallURL: function (url) {
-          if (url === null || url.length === 0) {
+          if (true) {
             extension.uninstallURL = null;
             return Promise.resolve();
           }
